spring-integration-zmq/src/main/java/org/springframework/integration/zmq/config/xml/ZmqMessageDrivenChannelAdapterParser.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.zmq.config.xml;
```

Comment:
The same naming concern for package name. I'd prefer `zeromq`

Please, consider to keep Copyright for all the new classes in the current year (2018) and update all other affected to the current one in its second year token